decidim-conferences/lib/decidim/conferences/admin_engine.rb

```diff
@@ -97,113 +97,131 @@ class AdminEngine < ::Rails::Engine
-          menu.item I18n.t("partners", scope: "decidim.admin.menu.conferences_submenu"),
-                    decidim_admin_conferences.conference_partners_path(current_participatory_space),
-                    if: allowed_to?(:read, :partner, conference: current_participatory_space),
-                    active: is_active_link?(decidim_admin_conferences.conference_partners_path(current_participatory_space))
-          menu.item I18n.t("conference_speakers", scope: "decidim.admin.menu.conferences_submenu"),
-                    decidim_admin_conferences.conference_speakers_path(current_participatory_space),
-                    if: allowed_to?(:read, :conference_speaker, conference: current_participatory_space),
-                    active: is_active_link?(decidim_admin_conferences.conference_speakers_path(current_participatory_space))
-
-          menu.item I18n.t("registrations", scope: "decidim.admin.menu.conferences_submenu"),
-                    "#",
-                    active: false, # is_active_link?(decidim_admin_conferences.conference_attachment_collections_path(current_participatory_space)),
-                    if: allowed_to?(:read, :conference_invite, conference: current_participatory_space) ||
-                        allowed_to?(:read, :registration_type, conference: current_participatory_space) ||
-                        allowed_to?(:read, :conference_registration, conference: current_participatory_space),
-                    submenu: { target_menu: :conferences_admin_registrations_menu }
-          menu.item I18n.t("conference_admins", scope: "decidim.admin.menu.conferences_submenu"),
-                    decidim_admin_conferences.conference_user_roles_path(current_participatory_space),
-                    if: allowed_to?(:read, :conference_user_role, conference: current_participatory_space),
-                    active: is_active_link?(decidim_admin_conferences.conference_user_roles_path(current_participatory_space))
-          menu.item I18n.t("moderations", scope: "decidim.admin.menu.conferences_submenu"),
-                    decidim_admin_conferences.moderations_path(current_participatory_space),
-                    if: allowed_to?(:read, :moderation, conference: current_participatory_space),
-                    active: is_active_link?(decidim_admin_conferences.moderations_path(current_participatory_space))
+          menu.add_item :edit_conference,
+                        I18n.t("info", scope: "decidim.admin.menu.conferences_submenu"),
+                        decidim_admin_conferences.edit_conference_path(current_participatory_space),
+                        position: 1,
+                        if: allowed_to?(:update, :conference, conference: current_participatory_space),
+                        active: is_active_link?(decidim_admin_conferences.edit_conference_path(current_participatory_space))
+          menu.add_item :components,
```

Comment:
```suggestion

          menu.add_item :components,
```

Reply:
my understanding is that you requested to add a new line.